schema-loader/README.md

```diff
@@ -258,18 +269,44 @@ $ java -jar scalardb-schema-loader-<version>.jar --jdbc -j <JDBC URL> -u <USER>
   }
 }
 ```
-- `compaction-strategy` should be `STCS`, `LCS` or `TWCS`. This is ignored in Cosmos DB and DynamoDB.
-- This `ru` value is set for all tables on this database even if `-r BASE_RESOURCE_UNIT` is set when Cosmos DB and DynamoDB. `ru` is ignored in Cassandra.
+
+You can also specify storage-specific options in the table definition as follows:
```

Comment:
```suggestion
You can also specify database/storage-specific options in the table definition as follows:
```
since we see `database-specific` in other places in the doc.